ClientSideEncryptionToServerSideEncryptionMigrationSamples/ClientSideKeyVaultKeyToCustomerProvidedKey/README.md

```diff
@@ -1,20 +1,23 @@
 # KeyVault Client Side Encryption Migration to Customer Provided Key Server Side Encryption
 
-This program functions as a migration of data in Azure Storage from client side encryption with KeyVault to server side encryption using a Customer Provided Key for server side encryption.
+This script functions as a sample of how to migrate data that was uploaded using client-side encryption with a local key to use server-side encryption with with a Customer-Provided Key. The sample includes a script to generate sample data, `create_sample_data.py`, as well as a script that will migrate all data within a given container, `miration.py`.
 
 ## Getting Started
 ### Prerequisites
-Requires installation of [Python](https://www.python.org/downloads/) and [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). Requires an [Azure subscription](https://azure.microsoft.com/en-us/free/) and an [Azure storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal).
+Requires installation of [Python 3](https://www.python.org/downloads/) and [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) (if you wish to programaticaly generate an encryption scope). Requires an [Azure subscription](https://azure.microsoft.com/en-us/free/) and an [Azure storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal).
```

Comment:
Nit: I think the spelling you are looking for is 'programmatically' (no worries, Google auto corrected me because I also spelt it wrong 😆